.taskcluster.yml

```diff
@@ -0,0 +1,169 @@
+# The version is always required
+version: 0
+# Top level metadata is always required
+metadata:
+  name: "TaskCluster Worker Tests"
+  description: "These tests should ensure that any new commits against taskcluster worker codebase are tested across all supported worker types in gekco."
+  owner: "{{ event.head.user.email }}" # the user who sent the pr/push e-mail will be inserted here
+  source: "{{ event.head.repo.url }}"  # the repo where the pr came from will be inserted here
+tasks:
+
+
+  ##########################################################
+  ##################### linux 64 build #####################
+  ##########################################################
+
+  - provisionerId: "{{ taskcluster.docker.provisionerId }}"
+    workerType: "{{ taskcluster.docker.workerType }}"
+    metadata:
+      name: "Build/test 64 bit taskcluster-worker on Debian GNU/Linux 8 (jessie)"
+      description: "This builds the 64 bit linux version of taskcluster-worker"
+      owner: "{{ event.head.user.email }}" # the user who sent the pr/push e-mail will be inserted here
+      source: "{{ event.head.repo.url }}"  # the repo where the pr came from will be inserted here
+    extra:
+      github:
+        # Events that will trigger this task
+        events:
+          - pull_request.*
```

Comment:
I've removed for now, we can discuss if we want to add it back in. I believe it would use the base revision of the `.taskcluster.yml` file rather than the head revision, so I don't think it is such a security concern, although someone could maliciously tie up our workers (although we could report them to github).